src/Domain.php

```diff
@@ -346,7 +374,7 @@ public function getPublicSuffix()
      */
     public function isResolvable(): bool
     {
-        return 1 < count($this->labels) && '.' !== substr($this->domain, -1, 1);
+        return 1 < count($this->labels ?? []) && '.' !== substr($this->domain, -1, 1);
```

Comment:
Why modifying this line labels is supposed to be an array and is defined in the constructor you should not have to do this .

Reply:
I 've got an exception at this place during library testing; on latest php versions count function should be used strictly on iterable/countable types,  but in some case with label manipulations (with/without label as i remember)  situation when $this->labels became null has been occured.

I will try to play back details of that case

Reply:
Okay maybe this should be fixed independently. Also this is why it is better to PR against the develop branch which is always ahead of the master branch. So maybe this was fixed I don't really remember